.github/workflows/publish-release.yaml

```diff
@@ -0,0 +1,33 @@
+name: Publish Release
+on: 
+  push:
```

Comment:
Ideally this runs only when we push a tags. I think it's safer to release only when we tag rather than when we push to `release`, as we might end up in situations where we need to re-sync the branch and that would trigger multiple publishing.

Side note: this job publishes to the "staging" repo. From there we need to manually "approve" the release so it's not a major problem if we accidentally push a wrong tag.

Reply:
I didn't mean to drop tags. I just suggested to add additional condition to trigger this workflow.

Reply:
> as we might end up in situations where we need to re-sync the branch and that would trigger multiple publishing.

Yeah but for this reason I would try to avoid multiple run of the same job on the same sha (like merge commit + pushed tag).

Reply:
Not sure if used the right quote